aggsender/aggsender.go

```diff
@@ -173,6 +173,9 @@ func (a *AggSender) Start(ctx context.Context) {
 
 	a.checkDBCompatibility(ctx)
 	a.checkInitialStatus(ctx)
+	if err := a.flow.CheckInitialStatus(ctx); err != nil {
+		a.log.Panicf("error checking flow Initial Status: %v, retrying in %s", err, a.cfg.DelayBeetweenRetries.String())
```

Comment:
There is no retrying so, just remove it:

```suggestion
		a.log.Panicf("error checking flow Initial Status: %w, err)
```